src/data/markdown/translated-guides/en/02 Using k6/05 Options.md

```diff
@@ -147,15 +147,15 @@ $ k6 run --no-connection-reuse --user-agent "MyK6UserAgentString/1.0" script.js
 ```
 
 ```bash
-c:\\k6> set "K6_NO_CONNECTION_REUSE=true" && set "K6_USER_AGENT=MyK6UserAgentString/1.0" && k6 run script.js
+C:\k6> set "K6_NO_CONNECTION_REUSE=true" && set "K6_USER_AGENT=MyK6UserAgentString/1.0" && k6 run script.js
 
-c:\\k6> k6 run --no-connection-reuse --user-agent "MyK6UserAgentString/1.0" script.js
+C:\k6> k6 run --no-connection-reuse --user-agent "MyK6UserAgentString/1.0" script.js
 ```
 
 ```bash
-c:\\k6> $env:K6_NO_CONNECTION_REUSE=true ; $env:K6_USER_AGENT="MyK6UserAgentString/1.0" ; k6 -run script.js
+PS C:\k6> $env:K6_NO_CONNECTION_REUSE=true; $env:K6_USER_AGENT="MyK6UserAgentString/1.0"; k6 -run script.js
```

Comment:
Sorry -  missed this one earlier:

```suggestion
PS C:\k6> $env:K6_NO_CONNECTION_REUSE="true"; $env:K6_USER_AGENT="MyK6UserAgentString/1.0"; k6 run script.js
```

Reply:
Nice catch - that's me having clumsy fingers.